### PR TITLE
The aim of this PR is to add support to Group Chats Topics that introduced in Bot API 6.3 

### PR DESCRIPTION
**Pull request by yurnov**
Bot API 6.3 (released November 5, 2022) added support for [Topics in Groups](https://telegram.org/blog/topics-in-groups-collectible-usernames#topics-in-groups) that is not supported currently (please find issue #1).

So, the aim of this PR is to add support for Topics in Group Chats with more than 100 members. It required an update of python-telegram-bot  to version [13.15](https://pypi.org/project/python-telegram-bot/13.15/) where [added full support for Bot API 6.3](https://docs.python-telegram-bot.org/en/stable/changelog.html)

Changes included in this PR:
- `python-telegram-bot` updated to 13.15;
- ~~version of poetry locked on 1.2.2 (the latest version that still uses old lock format, please find [changelog](https://github.com/python-poetry/poetry/blob/master/CHANGELOG.md) to keep maximum compatibility with your main branch) in Dockerfile and GitHub actions~~ Update: locked on `1.1.15`;
- optional configuration variable `THREAD_ID` added to set `message_thread_id` with support of `message_thread_id` added into the code;
- Annotations added in Dockerfile accordingly to [OCI Image Format Specification](https://github.com/opencontainers/image-spec/blob/main/annotations.md)
- Updated README.md

Proposed changes that are not done (will add as separate commits when you ok with that):
- add annotations `org.opencontainers.image.authors="Max Biloborodko<f1ashhimself@gmail.com>`

### CODE REVIEW

File: .github/workflows/app.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Install poetry
         run: |
-          pip install poetry==1.1.15
+          pip install poetry==1.2.2
```
Comment (F1ashhimself) [inline .github/workflows/app.yaml]: No poetry 1.2.x for this project :D It contains more bugs than you can even imagine, trust me :)

Please change the version and also you forgot to run `poetry update` to update lock file with new version of telegram bot.  Be aware that you are updating with correct version of poetry installed.

File: Dockerfile [hunk 2]
```diff
@@ -15,7 +20,7 @@ USER electricitybot
 WORKDIR /opt
 
 RUN export PATH="${PATH}":~/.local/bin/ && \
-    python -m pip install --no-cache-dir poetry && \
+    python -m pip install --no-cache-dir poetry==1.2.2 && \
```
Comment (F1ashhimself) [inline Dockerfile]: The same here, missed that in previous PR.

File: src/electricitybot/electricitybot.py [hunk 3]
```diff
@@ -50,7 +51,8 @@ def check_and_send(self):
         current_e_state = self.check_electricity()
         if self.previous_e_state != current_e_state:
             message = self.build_message(current_e_state)
-            self.tg_bot.send_message(chat_id=self.chat_id, text=message)
+            kwargs = dict(chat_id=self.chat_id, message_thread_id=self.thread_id, text=message)
+            self.tg_bot.send_message(**{k: v for k, v in kwargs.items() if v is not None})
```
Comment (F1ashhimself) [inline src/electricitybot/electricitybot.py]: Do not need to overcomplicate it `message_thread_id` parameter have default value `None` so we can just prepare it properly.

```suggestion
            self.tg_bot.send_message(chat_id=self.chat_id, message_thread_id=self.thread_id, text=message)
```

**Review by F1ashhimself — CHANGES_REQUESTED** [review]

File: .github/workflows/app.yaml (hunk 1 above)
Comment (yurnov) [reply .github/workflows/app.yaml]: Do you recommend keeping 1.1.x or to update to 1.3.1 with a new lock file format?

Comment (F1ashhimself) [reply .github/workflows/app.yaml]: Lets keep it 1.1.15 for now.

File: Dockerfile (hunk 2 above)
Comment (yurnov) [reply Dockerfile]: yep, I missed locking the poetry version too. Changed to `1.1.15`

**Review by F1ashhimself — APPROVED** [review]

File: .github/workflows/app.yaml (hunk 1 above)
Comment (yurnov) [reply .github/workflows/app.yaml]: Done. 

Probably I should update `README.md` with a `pip install poetry==1.1.15` as well.

Comment (F1ashhimself) [reply .github/workflows/app.yaml]: There are no need to do that. But probably it will be good to bump minor version. Lets do it next time :)

**Comment by yurnov** [comment]
Hi @F1ashhimself, I see that test failed [here](https://github.com/F1ashhimself/electricitybot/actions/runs/3698799953/jobs/6265435425)

**Comment by F1ashhimself** [comment]
@yurnov do not worry, I will take a look

**Comment by yurnov** [comment]
As I see it, test become [fail](https://github.com/yurnov/electricitybot/actions/runs/3698686246) starting from the commit 17205414aa326bffdfbb977b429c9f14f0c33897, probably we should revert it

**Comment by F1ashhimself** [comment]
No need, I will fix tests tomorrow morning.

**Pull request merged**